docs/content/node-operation/node-bootstrap.mdx

```diff
@@ -252,274 +227,69 @@ $ ./boot-tools/bootstrap check-machine-account --access-address access.mainnet.n
 
 This command will detect and provide information about common misconfigurations, or confirm that the machine account is configured correctly.
 
-## Step 3 - Getting Ready to Start your Flow Node
+## Step 3 - Start Your Flow Node
 
-<Callout type="danger">
+Ensure you have configured your node using the [Node Setup guide](/node-operation/node-setup).
 
-Before proceeding to Step 3, ensure that Step 1 was completed successfully with the bootstrap directory handy (default: `/var/flow/bootstrap`)
+### Confirming authorization
 
-</Callout>
-
-1. Expose TCP port **3569** on your firewall for both ingress and egress traffic
-
-If you are running Flow Access node, Collection node or an Execution node you will also need to expose TCP port **9000**.
+You can confirm your node's successful registration and authorization by reading the [Staking Contract](/core-contracts/staking-contract-reference#contract).
+At the end of the `EpochStaking` phase, the members of the Proposed Identity Table are confirmed as authorized participants in the next epoch. 
```

Comment:
```suggestion
At the end of the `STAKINGAUCTION` phase, the members of the Proposed Identity Table are confirmed as authorized participants in the next epoch. 
```
In the contract, it is referred to as staking auction, so I assume we want to keep calling it that here?

Reply:
I'll re-use the terminology from https://docs.onflow.org/staking/epoch-preparation/#phase-transitions:
* `Staking Auction Phase`
* `Epoch Setup Phase`
* `Epoch Committed Phase`

Reply:
Updated these in https://github.com/onflow/flow/pull/862/commits/29ba2f34aaf3d71d5badf5f8f5afac585deae3c1